test/create.test.js

```diff
@@ -33,9 +33,7 @@ describe('create', function() {
       price: {type: Number},
     }, {forceId: false});
 
-    db.once('connected', function() {
-      db.automigrate(done);
-    });
+    db.automigrate(done);
```

Comment:
I am why in some places you keep [`db.once(connected, cb)`](https://github.com/strongloop/loopback-connector-cloudant/pull/158/files#diff-811051bcaa8b24b437c9305920b029c7R21) and other places like this you remove it.